ui/raidboss/data/04-sb/trial/lakshmi.js

```diff
@@ -0,0 +1,338 @@
+      netRegexFr: NetRegexes.startsUsing({ id: '2485', source: 'Lakshmi', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '2485', source: 'ラクシュミ', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: '2485', source: '吉祥天女', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: '2485', source: '락슈미', capture: false }),
+      alertText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Vril + Knockback',
+          de: 'Vril + Rückstoß',
+          fr: 'Vril + Poussée',
+          ja: 'エーテル + 完全なる拒絶',
+          cn: '完全拒绝',
+          ko: '락슈미 에테르 + 넉백',
+        },
+      },
+    },
+    {
+      // Nobody with a different marker should be told to stack.
+      id: 'Lakshmi Headmarker Collect',
+      netRegex: NetRegexes.headMarker({ }),
+      run: (data, matches) => {
+        data.avoidStack = data.avoidStack || [];
+        if (matches.id !== '003E')
+          data.avoidStack.push(matches.target);
+      },
+    },
+    {
+      // Activating on any use of Hand of Beauty/Grace or Alluring Arm.
+      // Head markers don't appear until the end of the castbar,
+      // and every head marker section begins with one of these abilities,
+      // so this should be perfectly safe.
+      id: 'Lakshmi Headmarker Cleanup',
+      netRegex: NetRegexes.startsUsing({ id: ['2486', '2487', '2488'], source: 'Lakshmi', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: ['2486', '2487', '2488'], source: 'Lakshmi', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: ['2486', '2487', '2488'], source: 'Lakshmi', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: ['2486', '2487', '2488'], source: 'ラクシュミ', capture: false }),
+      netRegexCn: NetRegexes.startsUsing({ id: ['2486', '2487', '2488'], source: '吉祥天女', capture: false }),
+      netRegexKo: NetRegexes.startsUsing({ id: ['2486', '2487', '2488'], source: '락슈미', capture: false }),
+      run: (data) => delete data.avoidStack,
+    },
+    { // Stack marker
+      id: 'Lakshmi Pall of Light',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      delaySeconds: 0.5,
+      alertText: function(data, matches, output) {
+        if (!data.avoidStack.includes(data.me))
+          return;
+        return output.dontStack();
+      },
+      infoText: function(data, matches, output) {
+        if (data.avoidStack.includes(data.me))
+          return;
+        if (data.me === matches.target)
+          return output.stackOnYou();
+        return output.stack({ target: matches.target });
+      },
+      outputStrings: {
+        dontStack: {
+          en: 'Don\'t Stack!',
+        },
+        stackOnYou: {
+          en: 'Stack on YOU',
+          de: 'Stack auf DIR',
+          fr: 'Package sur VOUS',
+          ja: '自分に頭割り',
+          cn: '分摊点名',
+          ko: '쉐어징 대상자',
+        },
+        stack: {
+          en: 'Stack on ${target}',
+        },
```

Comment:
I think this is the general pattern, just in case any future find and replace operations take place. (Would require changing `target: matches.target` above to `player: matches.target`)
```suggestion
        stackOn: {
          en: 'Stack on ${player}',
          de: 'Sammeln auf ${player}',
          fr: 'Packez-vous sur ${player}',
          ja: '${player}と頭割り',
          cn: '靠近${player}分摊',
          ko: '"${player}" 쉐어징',
        },
```